client_plugin/drivers/shared/etcdops.go

```diff
@@ -255,9 +292,11 @@ func joinEtcdCluster(nodeAddr string, leaderAddr string, nodeID string) error {
 		"--initial-cluster-state", etcdClusterStateExisting,
 	}
 
+	// start the etcd cluster routine
```

Comment:
Join etcd cluster is called by manager nodes who are not initializing the cluster. To join, they must start their own etcd, hence the comment.